recipes/diffpy.pdfgui/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "diffpy.pdfgui" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/diffpy/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 4cf3106323544bbd6ead3a774c4dd1db534c618c116e0a0811a33493371150f5
+
+build:
+  # we need egg_dir to avoid overlapping diffpy/__init__.py files.
+  preserve_egg_dir: True
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  entry_points:
+    - pdfgui = diffpy.pdfgui.applications.pdfgui:main
+  osx_is_app: True
+
+requirements:
+  host:
+    - python =3.7
+    - pip
+    - setuptools
+
+  run:
+    - python =3.7
+    - setuptools
+    - six
+    - matplotlib-base
+    - wxpython =4.0.7
```

Comment:
Are these optional? I could not find them in the metadata.

Reply:
These are required dependencies to run the package. Where is the metadata to update this?

Reply:
See https://github.com/diffpy/diffpy.pdfgui/blob/v1.4.0/setup.py#L115-L119